install/database/86.sql

```diff
@@ -0,0 +1,6 @@
+INSERT INTO 'SETTING' ('settingid','setting','value','fieldType','helptext','options','cat','userChange','title','type','validation','ordering','default','userSee') VALUES
```

Comment:
Sorry about that, I had removed the value but the comment name.  Still learning but am getting their.